commands/drawCommand.js

```diff
@@ -22,14 +25,101 @@ export const RARITIES = [
   "Gold Ultra Rare",
 ];
 
+const storingDraws = async (message) => {
+  const response = await axios.get(
+    "https://db.ygoprodeck.com/api/v7/randomcard.php"
+  );
+  let highestRarityIndex = 0;
+  const card = response.data.card_sets;
+
+  card.forEach((element) => {
+    const cardSetRarity = element.set_rarity;
+    const cardSetRarityIndex = RARITIES.indexOf(cardSetRarity);
+    if (highestRarityIndex < cardSetRarityIndex) {
+      highestRarityIndex = cardSetRarityIndex;
+    }
+  });
+
+  message.reply(
+    `You just drew ${response.data.name}, a ${RARITIES[highestRarityIndex]} card`
+  );
+  const attachment = new MessageAttachment(
+    response.data.card_images[0].image_url
+  );
+
+  message.channel.send(attachment);
+
+  const ref = db.ref("users");
+  const userRef = ref.child(message.author.id);
+  userRef.update({
+    lastDrawTime: Date.now(),
+    name: message.author.username,
+  });
+
+  const inventoryRef = userRef.child("inventory");
+  inventoryRef.update({
+    [response.data.id]: {
+      name: response.data.name,
+      image: response.data.card_images[0].image_url,
+      price: response.data.card_prices[0].tcgplayer_price,
+      type: response.data.type,
+      rarity: RARITIES[highestRarityIndex],
+    },
+  });
+};
+
+const payForDraw = async (message) => {
+  const userRef = db.ref(`users/${message.author.id}`);
+  const snapshot = await userRef.once("value");
+  const userData = snapshot.val();
```

Comment:
this is something you do a lot, so you could make a `getUser()` function in `db.js` for example, and import and call that everywhere instead